### PR TITLE
Update TargetEncoding doc

### PR DESCRIPTION
**Pull request by hkvision**

### CODE REVIEW

**Comment by hkvision** [comment]
@yaxinchen666 Take a look.

File: pyzoo/zoo/friesian/feature/table.py [hunk 1]
```diff
@@ -1152,35 +1152,41 @@ def split(self, ratio, seed=None):
...
-               in the Table, then this column is used; otherwise, it is randomly generated with
-               range [0, kfold). Default is "__fold__".
-        :param drop_cat: Boolean. Drop the categorical columns if it is true. Default is True.
-        :param drop_folds: Boolean. Drop the fold column if it is true. Default is True.
+               in the FeatureTable, then this column is used; otherwise, it is randomly generated
+               within the range [0, kfold). Default is "__fold__".
+        :param drop_cat: boolean. Whether to drop the original categorical columns.
+               Default is False.
+        :param drop_fold: boolean. Drop the fold column if it is true. Default is True.
         :param out_cols: a nested list of str. Each inner list corresponds to the categorical
```
Comment (hkvision) [inline pyzoo/zoo/friesian/feature/table.py:1186]: @yaxinchen666 For out_cols, we should be the same as difference lag to support a single str or a list of str.

Comment (yaxinchen666) [reply pyzoo/zoo/friesian/feature/table.py:1186]: Got it.

**Comment by hkvision** [comment]
http://10.112.231.51:18888/job/ZOO-PR-Validation/6182/

**Pull request merged**